packages/netsuite-adapter/src/service_url/constant_urls.ts

```diff
@@ -0,0 +1,87 @@
+  scheduledscript: 'app/common/scripting/scriptlist.nl?scripttype=SCHEDULED&apiversion=&scriptfile=&bundlefilter=BLANK&sortcol=name&sortdir=ASC&csv=HTML&OfficeXML=F&pdf=&size=50&_csrf=AojhTU73ytHGABI4Ef4YZbn-QkSH0wGJjezesD8MUrTr5vLWA0g4lBIU62BZZ8Qi3mFJ8qcJSWu6b_uv7b4PJKg2WJf53xWJVGDeoBOBvE4Dim6B9fTCGCk-lqUEp8KSK4zXMfZJ3cZ9Gwhsh3la6p2BFu1TINH_rYuDqnkAFo4%3D&showall=F',
+  sdfinstallationscript: 'app/common/scripting/scriptlist.nl?scripttype=SDFINSTALLATION&apiversion=&scriptfile=&bundlefilter=BLANK&sortcol=name&sortdir=ASC&csv=HTML&OfficeXML=F&pdf=&size=50&_csrf=AojhTU73ytHGABI4Ef4YZbn-QkSH0wGJjezesD8MUrTr5vLWA0g4lBIU62BZZ8Qi3mFJ8qcJSWu6b_uv7b4PJKg2WJf53xWJVGDeoBOBvE4Dim6B9fTCGCk-lqUEp8KSK4zXMfZJ3cZ9Gwhsh3la6p2BFu1TINH_rYuDqnkAFo4%3D&showall=F',
+  usereventscript: 'app/common/scripting/scriptlist.nl?scripttype=USEREVENT&apiversion=&scriptfile=&bundlefilter=BLANK&sortcol=name&sortdir=ASC&csv=HTML&OfficeXML=F&pdf=&size=50&_csrf=AojhTU73ytHGABI4Ef4YZbn-QkSH0wGJjezesD8MUrTr5vLWA0g4lBIU62BZZ8Qi3mFJ8qcJSWu6b_uv7b4PJKg2WJf53xWJVGDeoBOBvE4Dim6B9fTCGCk-lqUEp8KSK4zXMfZJ3cZ9Gwhsh3la6p2BFu1TINH_rYuDqnkAFo4%3D&showall=F',
+  workflowactionscript: 'app/common/scripting/scriptlist.nl?scripttype=ACTION&apiversion=&scriptfile=&bundlefilter=BLANK&sortcol=name&sortdir=ASC&csv=HTML&OfficeXML=F&pdf=&size=50&_csrf=AojhTU73ytHGABI4Ef4YZbn-QkSH0wGJjezesD8MUrTr5vLWA0g4lBIU62BZZ8Qi3mFJ8qcJSWu6b_uv7b4PJKg2WJf53xWJVGDeoBOBvE4Dim6B9fTCGCk-lqUEp8KSK4zXMfZJ3cZ9Gwhsh3la6p2BFu1TINH_rYuDqnkAFo4%3D&showall=F',
+  portlet: 'app/common/scripting/scriptlist.nl?scripttype=PORTLET&apiversion=&scriptfile=&bundlefilter=BLANK&sortcol=name&sortdir=ASC&csv=HTML&OfficeXML=F&pdf=&size=50&_csrf=AojhTU73ytHGABI4Ef4YZbn-QkSH0wGJjezesD8MUrTr5vLWA0g4lBIU62BZZ8Qi3mFJ8qcJSWu6b_uv7b4PJKg2WJf53xWJVGDeoBOBvE4Dim6B9fTCGCk-lqUEp8KSK4zXMfZJ3cZ9Gwhsh3la6p2BFu1TINH_rYuDqnkAFo4%3D&showall=F',
+  restlet: 'app/common/scripting/scriptlist.nl?scripttype=RESTLET&apiversion=&scriptfile=&bundlefilter=BLANK&sortcol=name&sortdir=ASC&csv=HTML&OfficeXML=F&pdf=&size=50&_csrf=AojhTU73ytHGABI4Ef4YZbn-QkSH0wGJjezesD8MUrTr5vLWA0g4lBIU62BZZ8Qi3mFJ8qcJSWu6b_uv7b4PJKg2WJf53xWJVGDeoBOBvE4Dim6B9fTCGCk-lqUEp8KSK4zXMfZJ3cZ9Gwhsh3la6p2BFu1TINH_rYuDqnkAFo4%3D&showall=F',
+  suitelet: 'app/common/scripting/scriptlist.nl?scripttype=SCRIPTLET&apiversion=&scriptfile=&bundlefilter=BLANK&sortcol=name&sortdir=ASC&csv=HTML&OfficeXML=F&pdf=&size=50&_csrf=AojhTU73ytHGABI4Ef4YZbn-QkSH0wGJjezesD8MUrTr5vLWA0g4lBIU62BZZ8Qi3mFJ8qcJSWu6b_uv7b4PJKg2WJf53xWJVGDeoBOBvE4Dim6B9fTCGCk-lqUEp8KSK4zXMfZJ3cZ9Gwhsh3la6p2BFu1TINH_rYuDqnkAFo4%3D&showall=F',
+  center: 'app/common/custom/custcenters.nl',
+  centercategory: 'app/common/custom/custcategories.nl',
+  centerlink: 'app/common/custom/custtasks.nl',
+  centertab: 'app/common/custom/custsections.nl',
+  crmcustomfield: 'app/common/custom/eventcustfields.nl?whence=',
```

Comment:
Similar for other links...
```suggestion
  crmcustomfield: 'app/common/custom/eventcustfields.nl',
```